gitpod-io/gitpod
# decouple gp env from theia

#### What it does

- fix #3162: decouple gp env from theia

#### How to test

- test both Theia and VS Code
- get: should return exact and wildcard matches, should not return unmatched i.e. wildcards or other repos
- set/delete: should create, updated or delete based on exact matches, should not mutate unmatched i.e. wildcards or other repos
